etc/nginx/sites-enabled/git-http

```diff
@@ -1,4 +1,5 @@
-fastcgi_cache_path /etc/nginx/cache levels=1:2 keys_zone=gitserver_http:800m inactive=30m;
+fastcgi_cache_path /etc/nginx/cache-get levels=1:2 keys_zone=gitserver_http_get:200m inactive=30m;
+fastcgi_cache_path /etc/nginx/cache-post levels=1:2 keys_zone=gitserver_http_post:600m inactive=30m;
```

Comment:
@Rio517 Do you think 600m is enough here? Should we maybe go for 2-6g? One repo could already be 2g for an initial pull.

Reply:
The packed size is actually a lot smaller than we thought.  Our average repo, unpacked is like 100mb-200mb. The largest is 1.1GB, 2nd largest is 460mb.   Packed, that 460GB repo is only a 70MB download (at least as git clone shows). SO I think 600 is small, but maybe 2GB is good.  Also, keep in mind they expire quite fast.

Reply:
How fast you mean?

Reply:
And how it worked before with 800mb then? o_O

Reply:
Lots of keys got kicked out when they shouldn't have been!

Reply:
Btw, what do you think about the `inactive` option, guys? I don't know the precise business scenario, but I'd assume we can easily increase it for a few hours?

Reply:
That is a good call. We usually get bursts of activity around a specific cached response.  I think 2h would be good.